playbook/app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.jsx

```diff
@@ -17,15 +17,16 @@ type FixedConfirmationToastProps = {
   closeable?: boolean,
   data?: string,
   id?: string,
+  multiLine?: boolean,
   status?: "success" | "error" | "neutral" | "tip",
   text: string,
 }
 
 const FixedConfirmationToast = (props: FixedConfirmationToastProps) => {
   const [showToast, toggleToast] = useState(true)
-  const { className, closeable = false, status = 'neutral', text } = props
+  const { className, closeable = false, multiLine = false, status = 'neutral', text } = props
   const css = classnames(
-    `pb_fixed_confirmation_toast_kit_${status}`,
+    `pb_fixed_confirmation_toast_kit_${status}${multiLine === true ? '_multi_line' : ''}`,
```

Comment:
const css = classnames(
    `pb_fixed_confirmation_toast_kit_${status}`,
    'multi_line' : multiLine,
    globalProps(props),
    className
  )

Reply:
Thank you so much for your suggestions! 
I cleaned it up as you suggested, but used a ternary here: `multiLine ? '_multi_line' : null` Unless I'm missing something I think it does need the underscore.

Reply:
the `classnames()` function should be able to do that operation without the ternary operator interpolated in the string.

Here's an example from their docs that might be applicable:
`classnames('foo', { bar: true, duck: false }, 'baz', { quux: true }); // => 'foo bar baz quux'`

If not =, that's no big deal, I see you're trying to append it so it might not be able to accomplish exactly what you're going for.